spanner/ddl/ast.go

```diff
@@ -162,16 +162,17 @@ func (k Foreignkey) PrintForeignKey(c Config) string {
 // CreateTable encodes the following DDL definition:
 //     create_table: CREATE TABLE table_name ([column_def, ...] ) primary_key [, cluster]
 type CreateTable struct {
-	Name     string
-	ColNames []string             // Provides names and order of columns
-	ColDefs  map[string]ColumnDef // Provides definition of columns (a map for simpler/faster lookup during type processing)
-	Pks      []IndexKey
-	Fks      []Foreignkey
-	Comment  string
+	Name           string
+	ColNames       []string             // Provides names and order of columns
+	ColDefs        map[string]ColumnDef // Provides definition of columns (a map for simpler/faster lookup during type processing)
+	Pks            []IndexKey
+	Fks            []Foreignkey
+	InterleaveInto string //if not empty, this table will be interleaved
+	Comment        string
 }
 
 // PrintCreateTable unparses a CREATE TABLE statement.
-func (ct CreateTable) PrintCreateTable(config Config) string {
+func (ct CreateTable) PrintCreateTable(interleavedInto string, config Config) string {
```

Comment:
Why does PrintCreateTable get interleavedInfo as an arg?
Shouldn't it just use ct.InterleavedInto?

Reply:
makes sense, updated PrintCreateTable function